src/utilities/assertValidName.js

```diff
@@ -28,12 +29,13 @@ export function assertValidName(
   if (!isIntrospection && name.slice(0, 2) === '__' && !hasWarnedAboutDunder) {
     hasWarnedAboutDunder = true;
     /* eslint-disable no-console */
-    if (console && console.error) {
+    if (console && console.warn) {
       const error = new Error(
         `Name "${name}" must not begin with "__", which is reserved by ` +
-        'GraphQL introspection.'
+        'GraphQL introspection. In a future release of graphql this will ' +
```

Comment:
I went with `graphql` because that is the NPM package name. `graphql-js` is only the name of the GitHub repo.